src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/SqlManagerController.php

```diff
@@ -173,17 +174,21 @@ public function processFormAction(Request $request)
      */
     public function createAction(Request $request)
     {
-        $data = $this->getSqlRequestDataFromRequest($request);
+        try {
+            $data = $this->getSqlRequestDataFromRequest($request);
 
-        $sqlRequestForm = $this->getSqlRequestFormBuilder()->getForm($data);
-        $sqlRequestForm->handleRequest($request);
+            $sqlRequestForm = $this->getSqlRequestFormBuilder()->getForm($data);
+            $sqlRequestForm->handleRequest($request);
 
-        $result = $this->getSqlRequestFormHandler()->handle($sqlRequestForm);
+            $result = $this->getSqlRequestFormHandler()->handle($sqlRequestForm);
 
-        if (null !== $result->getIdentifiableObjectId()) {
-            $this->addFlash('success', $this->trans('Successful creation.', 'Admin.Notifications.Success'));
+            if (null !== $result->getIdentifiableObjectId()) {
+                $this->addFlash('success', $this->trans('Successful creation.', 'Admin.Notifications.Success'));
```

Comment:
```suggestion
                $this->addFlash('success', $this->trans('Successful creation', 'Admin.Notifications.Success'));
```

Reply:
I'm sorry, it's not related to the PR, I didn't change any wording :(

Reply:
I confirm, @PierreRambaud only changed the line indentation 😉

Reply:
Hi, I'm removing the 'waiting for wording' label then